src/Robot-Dobot_MG400/main.py

```diff
@@ -195,10 +203,21 @@ def deinitialize(self):
 
     def configure(self):
 
+        self.movement_height = float(self.movement_height)
         self.set_acceleration_linear(self.acceleration_factor)  # Linear acceleration factor 1-100
         self.set_speed_global(self.global_speed_factor)  # Global speed factor 1-100
         self.set_speed_linear(self.speed_factor)  # Linear speed factor 1-100
-        self._last_xyzr = (None, None, None, None)
+        # self._last_xyzr = (None, None, None, None)
+        self._last_xyzr = self.get_position()
+
+    def unconfigure(self):
+
+        if self.use_jump_mode:
+            self.move_linear(self._last_xyzr[0], self._last_xyzr[1], self.movement_height, self._last_xyzr[3])
+            self.sync(self.reach_position_timeout)
+
+        mode = self.get_robot_mode()
+        # todo: check for errors
```

Comment:
Should the error checking be implemented before going to test the driver?

And secondly, checking for errors when the measurement is already over might be a bit late.

Reply:
This line is already part of the productive driver version. `self.get_robot_mode` is not needed at the end of the driver. I think I used it once to check whether the robot is still responsive when there have been errors in configure during enabling the robot. In this case, one could have done some checks to present an error message.
Because the code it not used, I removed it